projects/fsastorefrontlib/src/cms-components/my-account/premium-calendar/premium-calendar.component.html

```diff
@@ -1,238 +1,134 @@
-            active: policy.opened
-          }"
-        >
-          <div class="col">
-            <div class="row">
-              <div class="col-4">
-                <div class="image-wrapper">
-                  <img
-                    class="image-item"
-                    [src]="
-                      getBaseUrl() +
-                      policy.mainProduct.coverageProduct.images[0]?.url
-                    "
-                    alt=""
-                  />
+<ng-container *ngIf="policies$ | async as policies; else noPolicies">
+  <ng-container *ngIf="policiesLoaded$ | async; else loading">
+    <ng-container
+      *ngIf="policies?.insurancePolicies?.length > 0; else noPolicies"
+    >
+      <div class="container-fluid">
+        <!-- Desktop Layout-->
+        <div class="section-header-heading row d-none d-lg-flex text-left">
+          <h5 class="col text-center">{{ 'policy.policy' | cxTranslate }}</h5>
+          <h5 class="col">{{ 'fscommon.paymentFrequency' | cxTranslate }}</h5>
+          <h5 class="col">{{ 'premiumCalendar.dueDate' | cxTranslate }}</h5>
+          <h5 class="col">{{ 'fscommon.premium' | cxTranslate }}</h5>
+          <h5 class="col">{{ 'fscommon.paymentMethod' | cxTranslate }}</h5>
+        </div>
+        <div *ngFor="let policy of policies.insurancePolicies; let i = index">
+          <div
+            class="premium-data-row row border boder-color-3"
+            (click)="toggleActiveAccordion(i)"
+            [ngClass]="{
+              clickable: policy.billingData?.length > 1,
+              active: selectedIndexes.includes(i)
+            }"
+          >
+            <div class="col">
+              <div class="row">
+                <div class="col-4">
+                  <div class="image-wrapper small">
+                    <img
+                      class="image-item"
+                      [src]="
+                        getBaseUrl() +
+                        policy.mainProduct.coverageProduct.images[0]?.url
+                      "
+                      alt=""
+                    />
+                  </div>
+                </div>
+                <div class="col-8 mw-100">
+                  {{ policy.categoryData?.name }} <br />
+                  {{ policy.contractNumber }}
                 </div>
               </div>
-              <div class="col-8">
-                {{ policy.categoryData?.name }} <br />
-                {{ policy.contractNumber }}
+            </div>
+            <div class="col">
+              <h5 class="d-lg-none m-0">
+                {{ 'fscommon.paymentFrequency' | cxTranslate }}
+              </h5>
+              {{ policy.paymentFrequency }}
+            </div>
+            <div
+              class="col text-left"
+              [ngClass]="
+                policy.billingData[0]?.paymentStatus == 'Overdue'
+                  ? 'warn'
+                  : 'notice'
+              "
+            >
+              <h5 class="d-lg-none p-0 m-0">
```

Comment:
I would leave out `p-0` and `m-0`, since padding here does nothing in the first place and margin is the default h5 margin, so it shouldn't look a lot different with it. If, however, it is really spaced out then maybe consider using span or or a div instead of h5 and add some heading or title class to it... Same goes for the `<h5 class="d-lg-none m-0">`